py/torch_tensorrt/dynamo/refitting/refit_engine.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
```

Comment:
Move to examples/dynamo: 
ex. https://github.com/pytorch/TensorRT/blob/main/examples/dynamo/custom_kernel_plugins.py
(rendered version) https://pytorch.org/TensorRT/tutorials/_rendered_examples/dynamo/custom_kernel_plugins.html#wrapping-custom-kernels-to-use-in-tensorrt
https://github.com/pytorch/TensorRT/blob/a8a0797158fcfb8263ad719be8c7bf27f10fd5af/docsrc/index.rst?plain=1#L114
(use: sphinx-gallery)